src/detail/connection_handle.cpp

```diff
@@ -35,6 +35,31 @@ namespace sqlpp
   {
     namespace detail
     {
+      namespace
```

Comment:
We are in `::sqlpp::mysql::detail` already.
Why the extra anonymous namespace here?